sdk/dapp-kit/src/components/WalletProvider.tsx

```diff
@@ -33,6 +36,9 @@ type WalletProviderProps = {
 	/** The key to use to store the most recently connected wallet account. */
 	storageKey?: string;
 
+	/** The theme to use for styling UI components. Defaults to using the light theme. */
+	theme?: Theme | null;
```

Comment:
I figured I'd add this to the wallet provider and add the `null` option if people want to opt out of using the UI components, I'm not sure if that's the best way to go about but it seems to make sense for now?

Reply:
I wonder if we can do dynamic injection through something like context to make it just inject when components are used instead of `theme={null}`. I think the API here is pretty unintuitive so want to find another way to do this.

Reply:
yeah good call, let me take a stab at this

Reply:
actually, would you be cool if I leave a TODO to come back to this? I'd like to get the other styling/theming work landed ASAP